### DISCUSSION TITLE
Did it counts as "wached" when you dl the video?

### DISCUSSION THREAD

**Discussion by rrondo** [jely2002/youtube-dl-gui#474] [General]
Does the system somehow count these "views" ? Does the author of YouTube video get any benefits from you, who just downloaded his video without even opening it?

Thanks!

**Comment by jely2002** [comment]
This can occasionally happen yes! The way yt-dlp downloads can be similar to watching it which can mean YouTube may show you part of the video was watched.

Creators don't get any benefits as ads are not displayed of course. So I'd always recommend supporting creators you like another way or watching with ads if possible.
